eclair-core/src/main/scala/fr/acinq/eclair/db/pg/PgAuditDb.scala

```diff
@@ -314,6 +327,8 @@ class PgAuditDb(implicit ds: DataSource) extends AuditDb with Logging {
         statement.setBoolean(6, m.isMultiPart)
         statement.setString(7, m.experimentName)
         statement.setString(8, m.recipientNodeId.value.toHex)
+        statement.setString(9, m.paymentHash.toHex)
+        statement.setString(10, serialization.write(m.routingHints_opt))
```

Comment:
For consistency with migrated data what we really want is a `null` value when there are no routing hints:

```suggestion
        statement.setString(10, m.routingHints_opt.map(serialization.write(_)).orNull)
```

That's what we do for optional channel updates in the network db:
https://github.com/ACINQ/eclair/blob/44e71277a6f1f80d28c1c14cee2f974eb2e1a6db/eclair-core/src/main/scala/fr/acinq/eclair/db/pg/PgNetworkDb.scala#L111-L112

Reply:
Done.